dom.go

```diff
@@ -52,11 +52,19 @@ type Copier interface {
 	Copy() Component
 }
 
+// Mounter is an optional interface that a Component can implement in order
+// to receive component mount events.
+type Mounter interface {
+	// Mount is called after the component has been mounted, after the DOM node
+	// has been attached.
```

Comment:
replace `, after the DOM node` with `, i.e. after the DOM node` since we're explaining what it means for a component to be mounted. There isn't an important distinction between a component being mounted and a DOM node being attached.